packages/flutter_test/lib/src/binding.dart

```diff
@@ -1258,6 +1275,12 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
   /// directly to run (either by using [WidgetTester.pumpBenchmark] or invoking
   /// [Window.onBeginFrame] and [Window.onDrawFrame]).
   ///
+  /// This allows all frame requests from the engine to be serviced, and allows
+  /// all frame requests that are artificially triggered to be serviced, but
+  /// prevents the framework from requesting any frames from the engine itself.
```

Comment:
Nit: is `ignores [scheduleFrame] requests from the framework` more accurate than `prevents the framework from requesting any frames from the engine itself.`?

Reply:
`prevents the framework from requesting any frames from the engine itself.` is the original version in `LiveTestWidgetsFlutterBinding.framePolicy` doc I moved from. I agree the new version but will leave this conversation un-resolved for visibility.